opentelemetry-dynatrace/src/exporter/mod.rs

```diff
@@ -0,0 +1,53 @@
+//! OpenTelemetry Dynatrace Exporter Configuration
+#![allow(unused_attributes)]
```

Comment:
Any special reason why we should allow this here?

Reply:
I remember cargo fmt throwing some warnings in previous versions about unused variables, but I guess this is now obsolete. I removed the configuration.